### PR TITLE
adding support for claims with null values

### PR DESCRIPTION
**Pull request by tushargupta51**
#136 

@brentschmaltz @polita

### CODE REVIEW

File: src/System.IdentityModel.Tokens.Jwt/JwtPayload.cs [hunk 1]
```diff
@@ -534,11 +536,19 @@ internal static object GetClaimValueUsingValueType(Claim claim)
                 return JArray.Parse(claim.Value);
             }
 
+            if (claim.ValueType == JsonClaimValueTypes.JsonNull)
```
Comment (polita) [inline src/System.IdentityModel.Tokens.Jwt/JwtPayload.cs]: This whole block is looking rife for a switch statement, but we don't need to change it right now.

**Comment by polita** [comment]
LGTM

File: test/System.IdentityModel.Tokens.Jwt.Tests/JwtPayloadTest.cs [hunk 2]
```diff
@@ -138,9 +138,14 @@ public void FirstClassProperties()
         [Fact]
         public void TestClaimWithNullValue()
         {
-            JwtPayload jwtPayload = new JwtPayload();
-            jwtPayload.Add("testClaim", null);
-            List<Claim> claims = jwtPayload.Claims as List<Claim>;   // this should not throw
+            JwtPayload payload = new JwtPayload();
```
Comment (brentschmaltz) [inline test/System.IdentityModel.Tokens.Jwt.Tests/JwtPayloadTest.cs]: I would like to see test that round-trips to and from JSON.

**Comment by brentschmaltz** [comment]
:shipit:

**Pull request merged**